### ISSUE TITLE
Incorrect use of <aside> in Toolbar

### ISSUE THREAD

**Issue by benborgers** [carbon-design-system/ibm-security#711]
## Bug - the `aside` element cannot have `role="menu"`

Accessibility testers pointed out that the `aside` HTML element may not have `role="menu"` (the allowed roles are listed [in the first row of this table](https://fae.disability.illinois.edu/rulesets/ROLE_7/)). 

Looking into it further, I think the issue lies with the `aside` element, not the `role="menu"`. [This answer](https://stackoverflow.com/a/40399165) advises against it, saying: 

> `aside` is for "tangentially related" content, but the primary navigation is hardly only "tangentially" related to the document.

---

Here's the part of the `Toolbar` component: 

https://github.com/carbon-design-system/ibm-security/blob/66ab719ee54ed957adb1c791c464894d829ea569/src/components/Toolbar/Toolbar.js#L250-L280

### Optional information

**Version -** `1.28.0`
